src/instructlab/taxonomy/diff.py

```diff
@@ -71,7 +71,7 @@ def diff(ctx, taxonomy_path, taxonomy_base, yaml_rules, quiet):
             for f in updated_taxonomy_files:
                 click.echo(f)
     try:
-        read_taxonomy(None, taxonomy_path, taxonomy_base, yaml_rules)
+        validate_taxonomy(None, taxonomy_path, taxonomy_base, yaml_rules)
     except (SystemExit, yaml.YAMLError) as exc:
```

Comment:
(No action required) Insterestingly, we don't seem to catch `TaxonomyReadingException` that the function can raise.

Reply:
It's also unclear which exception the `SystemExit` catch is supposed to handle, since the one that (was) raised from `get_documents` is already converted into `TaxonomyReadingException`.